apps/aefate/src/aefa_engine_state.erl

```diff
@@ -100,9 +123,22 @@
 -include_lib("aecontract/include/aecontract.hrl").
 -include_lib("aecontract/include/hard_forks.hrl").
 
+-type debugger_status() :: break | next | step | continue.
+-type debugger_location() :: none | {string(), integer()}.
+
 -type void_or_fate() :: ?FATE_VOID | aeb_fate_data:fate_type().
 -type pubkey() :: <<_:256>>.
 
+-record(debug_info, { status = continue       :: debugger_status()
+                    , location = none         :: debugger_location()
+                    , breakpoints = []        :: list()
+                    , current_instruction = 0 :: integer()
+                    , vars_registers = #{}    :: map()
```

Comment:
You can specify types of keys and values of a map.

Reply:
Done here 1205b532673694f7d341bd59acae9e47a4e809f5